src/lib.rs

```diff
@@ -8,6 +8,7 @@ pub mod bits;
 pub mod chunkscan;
 #[cfg(feature = "cli")]
 pub mod cli;
+pub mod crypto;
```

Comment:
Do we have another "crypto" module that we might add here?  ~~"nft" perhaps?~~ I can't think of anything that fits.

Reply:
eh maybe not, lemme lift hpke module